packages/@aws-cdk/pipelines/README.md

```diff
@@ -62,6 +107,524 @@ class MyApplication extends Stage {
+Sources objects can be created by factory methods on the `CodePipelineSource` class:
+
+* `CodePipelineSource.gitHub('org/repo', { branch, authentication })`: use a
+  GitHub respository as the source. By default, it assumes your GitHub OAuth
+  token lives in a Secret in Secrets Manager with the name `github-token`.
+  You can override that by passing the `authentication` property.
+
+#### Additional inputs
+
+`ScriptStep` allows passing in more than one input: additional
+inputs will be placed in the directories you specify. Any step that produces an
+output file set can be used as an input, such as a `CodePipelineSource`, but
+also other `ScriptSteps`:
+
+```ts
+const prebuild = new ScriptStep('Prebuild', {
+  input: CodePipelineSource.gitHub('myorg/repo1'),
+  primaryOutputDirectory: './build',
+  commands: ['./build.sh'],
+});
+
+const synth = new ScriptStep('Synth', {
+  input: CodePipelineSource.gitHub('myorg/repo2'),
+  additionalInputs: {
+    'subdir': CodePipelineSource.gitHub('myorg/repo3'),
+    '../siblingdir': prebuild,
+  },
+
+  commands: ['./build.sh'],
+});
+
+const pipeline = new CodePipeline(this, 'Pipeline', {
+  synth,
+});
+```
+
+### CDK application deployments
+
+After you have defined the pipeline and the `synth` step, you can add one or
+more CDK `Stages` which will be deployed to their target environments. To do
+so, call `pipeline.addStage()` on the Stage object:
+
+```ts
+// Do this as many times as necessary with any account and region
+// Account and region may different from the pipeline's.
+pipeline.addStage(new MyApplicationStage(this, 'Prod', {
+  env: {
+    account: '123456789012',
+    region: 'eu-west-1',
+  }
+}));
+```
+
+CDK Pipelines will automatically discover all `Stacks` in the given `Stage`
+object, determine their dependency order, and add appropriate actions to the
+pipeline to publish the assets referenced in those stacks and deploy the stacks
+in the right order.
+
+#### Deploying in parallel
+
+By default, all applications added to CDK Pipelines by calling `addStage()` will
+be deployed in sequence, one after the other. If you have a lot of stages, you can
+speed up the pipeline by choosing to deploy some stages in parallel. You do this
+by calling `addWave()` instead of `addStage()`: a *wave* is a set of stages that
+are all deployed in parallel instead of sequentially. Waves themselves are still
+deployed in sequence. For example, the following will deploy two copies of your
+application to `eu-west-1` and `eu-central-1` in parallel:
+
+```ts
+const europeWave = pipeline.addWave('Europe');
+europeWave.addStage(new MyApplicationStage(this, 'Ireland', {
+  env: { region: 'eu-west-1' }
+}));
+europeWave.addStage(new MyApplicationStage(this, 'Germany', {
+  env: { region: 'eu-central-1' }
+}));
+```
+
+#### Deploying to other accounts / encrypting the Artifact Bucket
+
+CDK Pipelines can transparently deploy to other Regions and other accounts
```

Comment:
```suggestion
CDK Pipelines can transparently deploy to other regions and other accounts
```

Reply:
Well I can un-capitalize but I'm fairly sure communication guidelines are going to say the word "Region" has to be capitalized everywhere. This is at least one of the transformations that will consistently get done to every blog post you write :).